vocabularies/scripts/update-csv.py

```diff
@@ -0,0 +1,76 @@
+# This script updates the keys in a given file with the provided values.
+#
+# Depends on Python 3
+#
+# Install dependencies:
+#   pip install -r requirements.txt
+#
+#
+#   Invocation:
+#   cd vocabularies/scripts
+#   python update-csv.py locations spaghetti.csv
+#
+#   Params:
+#    - [which]: Name of csv file to update
+#               (e.g. 'recapCustomerCodes', 'organizations').
+#    - [updateInfo] Name of csv file with values to update
+import sys
+import csv
+
+
+def csv_to_dict(file_name: str) -> dict:
+    information_dict = {}
+    tmp_dict = {}
+
+    try:
+        # Open the CSV file
+        with open(file_name, 'r') as f:
+            raw = f.readlines()
```

Comment:
raw_csv probably reads a bit better here so we know what we are working with down below.